mods/ts/weapons/healweapons.yaml

```diff
@@ -1,25 +1,25 @@
-Heal:
+^HealWeapon:
 	ReloadDelay: 80
 	Range: 2c849
 	Report: healer1.aud
-	ValidTargets: Infantry
-	Projectile: Bullet
-		Speed: 1c682
+	Projectile: InstantHit
 	Warhead@1Dam: SpreadDamage
```

Comment:
Or even better IMO would be a compromise to implement `TargetDamageWarhead` that only supports `TargetType.Actor`, with a TODO for the target scanning.  I'm assuming that the heal weapons restrict targeting to actors, so don't need the extra logic.